test/unit/tasks/yarn-install-spec.js

```diff
@@ -12,6 +12,10 @@ const modulePath = '../../../lib/tasks/yarn-install';
 const errors = require('../../../lib/errors');
 
 describe('Unit: Tasks > yarn-install', function () {
+    beforeEach(() => {
+        process.env = {};
```

Comment:
Do we have any tests that depend on `NODE_ENV=test` (or something like that)

Reply:
Not that I'm aware of - the only NODE_ENV usage I recall seeing was when we set `NODE_ENV = 'production'` when calling `yarn install `, and in that case it's a hardcoded value.